app/views/otus/batch_load/castor/_batch_load.html.erb

```diff
@@ -0,0 +1,35 @@
+<div>
+  <h2> Castor batch load </h2>
```

Comment:
I think it might be worth renaming this.  It's generic enough that it's not just Castor specific.  "OTU with identifier" or something?